frontend/scripts/pages/data.page.jsx

```diff
@@ -2,24 +2,31 @@
 import DataMarkup from 'html/data.ejs';
 import NavMarkup from 'html/includes/_nav.ejs';
 import FooterMarkup from 'html/includes/_footer.ejs';
-import AutocompleteCountriesMarkup from 'html/includes/_autocomplete_countries.ejs';
 import FeedbackMarkup from 'html/includes/_feedback.ejs';
-import DataContentContainer from 'containers/data/data-content.container';
 import NavContainer from 'containers/shared/nav.container';
 import { loadContext } from 'actions/data.actions';
 import 'styles/data.scss';
 import 'styles/components/shared/veil.scss';
 import 'styles/components/shared/modal.scss';
+import DataPortalPage from 'containers/data/data-portal-page.container';
+import { render } from 'react-dom';
+import React from 'react';
+import { Provider } from 'react-redux';
 
 export const mount = (root, store) => {
   root.innerHTML = DataMarkup({
     nav: NavMarkup({ page: 'data' }),
     footer: FooterMarkup(),
-    autocomplete_countries: AutocompleteCountriesMarkup(),
     feedback: FeedbackMarkup()
   });
-  new DataContentContainer(store);
-  store.dispatch(loadContext());
 
+  render(
+    <Provider store={store}>
+      <DataPortalPage />
+    </Provider>,
+    document.querySelector('.c-container')
+  );
+
+  store.dispatch(loadContext());
```

Comment:
I moved this to a `.thunks.js` file. Let's strive to maintain the `.page.js` files as dumb as possible.